src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,43 @@
+package study;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+    @Test
+    void splitTest() {
+        String[] result1 = "1,2".split(",");
+        String[] result2 = "1".split(",");
+
+        assertThat(result1).contains("1", "2");
+        assertThat(result2).containsExactly("1");
+    }
+
+    @Test
+    void substringTest() {
+        String exmaple = "(1,2)";
+        String result = exmaple.substring(exmaple.indexOf("(") + 1, exmaple.indexOf(")"));
+
+        assertThat(result).isEqualTo("1,2");
+    }
+
+    @DisplayName("string의 charAt을 호출할 때 배열의 범위를 벗어나면 StringIndexOutOfBoundsException이 발생한다")
+    @Test
+    void carAtTest() {
+        String example = "abc";
+        char first = 'a';
+        char second = 'b';
+        char third = 'c';
+
+        assertThat(first).isEqualTo(example.charAt(0));
+        assertThat(second).isEqualTo(example.charAt(1));
+        assertThat(third).isEqualTo(example.charAt(2));
```

Comment:
테스트를 하려는 대상은 charAt(0)이지 first가 아닙니다. 
```suggestion
        assertThat(example.charAt(0)).isEqualTo(first);
        assertThat(example.charAt(1)).isEqualTo(second);
        assertThat(example.charAt(2)).isEqualTo(third);
```